Wont Compile

Using CLI.

command used:  mbed compile -t GCC_ARM -m UBLOX_EVK_ODIN_W2

Upon Compiling for ODIN:

[ERROR] In file included from ./esp8266-driver/ESP8266Interface.cpp:18:0:
./esp8266-driver/ESP8266Interface.h:20:41: fatal error: network-socket/NetworkStack.h: No such file or directory
 #include "network-socket/NetworkStack.h"

--- a/esp8266-driver.lib
+++ b/esp8266-driver.lib
@@ -1,1 +1,1 @@
-https://github.com/armmbed/esp8266-driver/#29d63ae2ee0a233e2fbd9577cdddc7661bb783d1
+https://github.com/armmbed/esp8266-driver/#dc37b65ca877d969aa492f348626df6e1b0b1df0

